app/timer.js

```diff
@@ -32,26 +33,26 @@ define(['jquery'], function($) {
     seconds = time%60;
```

Comment:
Use a function. You might want to define some module time for conversions, as you convert between seconds and milliseconds in several places.